packages/manager/src/features/Billing/InvoiceDetail/InvoiceTable.tsx

```diff
@@ -38,12 +40,123 @@ interface Props {
   loading: boolean;
 }
 
-const InvoiceTable = (props: Props) => {
+export const InvoiceTable = (props: Props) => {
   const { classes } = useStyles();
+  const MIN_PAGE_SIZE = 25;
   const flags = useFlags();
+  const columns = flags.dcSpecificPricing ? 9 : 8;
```

Comment:
```suggestion
  const NUM_COLUMNS = flags.dcSpecificPricing ? 9 : 8;
```
Optional rename for clarity.